scripts/utils.py

```diff
@@ -0,0 +1,38 @@
+inverse = ['Australia', 'Euro', 'Ireland', 'New Zealand', 'United Kingdom']
```

Comment:
Let's delete this file

Reply:
`utils.py` is deleted and variables moved into `process.py` script